### PR TITLE
Define FCP where first used

### PR DESCRIPTION
**Pull request by mark-i-m**
Resolves #507

### CODE REVIEW

File: src/walkthrough.md [hunk 1]
```diff
@@ -120,13 +120,13 @@ itself to reflect the course of the discussion (e.g. new alternatives or prior
...
 In the end, when the discussion seems to reach a consensus and die down a bit,
-a rust team member may propose to move to FCP with one of three possible dispositions.
-This means that they want the other members of the appropriate teams to review
-and comment on the RFC. More discussion may ensue, which may result in more changes
-or unresolved questions being added. At some point, when everyone is
-satisfied, the RFC enters the "final comment period" (FCP), which is the last
-chance for people to bring up objections. When the FCP is over, the disposition is
-adopted. Here are the three possible dispositions:
+a rust team member may propose to move to "final comment period" (FCP) with one
+of three possible dispositions.  This means that they want the other members of
```
Comment (JohnTitor) [inline src/walkthrough.md]: unnecessary whitespace is here
```suggestion
of three possible dispositions. This means that they want the other members of
```

**Comment by mark-i-m** [comment]
@JohnTitor Lol, accepting your suggestion unapproved the PR :P Could you reapprove please?

**Comment by mark-i-m** [comment]
Wow, you're fast!
